velox/common/memory/SharedArbitrator.cpp

```diff
@@ -63,29 +63,31 @@ std::string memoryPoolAbortMessage(
       << victim->treeMemoryUsage();
   return out.str();
 }
-} // namespace
 
-SharedArbitrator::SharedArbitrator(const MemoryArbitrator::Config& config)
-    : MemoryArbitrator(config), freeCapacity_(capacity_) {
-  RECORD_METRIC_VALUE(kMetricArbitratorFreeCapacityBytes, freeCapacity_);
-  VELOX_CHECK_EQ(kind_, config.kind);
-}
-
-std::string SharedArbitrator::Candidate::toString() const {
-  return fmt::format(
-      "CANDIDATE[{} RECLAIMABLE[{}] RECLAIMABLE_BYTES[{}] FREE_BYTES[{}]]",
-      pool->root()->name(),
-      reclaimable,
-      succinctBytes(reclaimableBytes),
-      succinctBytes(freeBytes));
+std::vector<SharedArbitrator::Candidate> getCandidateStats(
```

Comment:
Is there any logic change in this function?

Reply:
No logic change in this function.